__tests__/apps.test.js

```diff
@@ -47,4 +48,32 @@ describe('app', () => {
                 });
             });
         });
+    describe('GET/api/reviews', () => {
+            test('200: returns a array of games reviews as objects', () => {
+                return request(app)      
+                .get('/api/reviews')
+                .expect(200)
+                .then(({ body }) => {
+                    expect(body.reviews.length).toBe(13)
+                    body.reviews.forEach((review) => {
+                        expect(review).toEqual(
+                            expect.objectContaining({
+                            review_id: expect.any(Number),
+                            title: expect.any(String),
+                            designer: expect.any(String),
+                            owner: expect.any(String),
+                            review_img_url: expect.any(String),
+                            review_body: expect.any(String),
+                            category: expect.any(String),
+                            created_at: expect.any(String),
+                            votes: expect.any(Number)
+                        }));
+                    });
+                    expect(body.reviews).toBeSortedBy('created_at', {
+                        descending : true,
+                        coerce : false
```

Comment:
Should not need to coerce since they're all the same data type... does this fail without this line?
If not, you can remove it

Reply:
Looks like it doesn't fail actually! I'll remove it now.